wiki/Welcome/en.md

```diff
@@ -0,0 +1,31 @@
+# Welcome
+
+Welcome to _osu!_, a free-to-win rhythm game—developed by peppy—with four game modes: osu!standard, a circle clicking simulator; osu!taiko, a drum drumming emulator; osu!catch, a fruit salad catcher; osu!mania, a key smashing mania.
```

Comment:
mixed old one(last commit) and current one?
current is a bit short. and i think old ver is more attaching, since it tells visitors the truth that osu! is very popular around the world :p

Reply:
no, I overwrote the changes
- I suppose 7a84ea0 is a bit more attaching, but it exceeds the 250 character limit
  - it has 292 characters (or 274, without the markdown syntax)
  - this one has 233 characters (or 231, without the markdown syntax)
- I think 7a84ea0 also has too much information... 
<!-- 
damn you webhooks!
  - yes, it has millions of players
  - yes, it "has" thousands of songs
     - note: has = possess, own, or hold... those songs don't belong to osu!, well most of them
-->